.pylintrc

```diff
@@ -4,6 +4,9 @@ good-names=id,tb
 [LOGGING]
 logging-format-style=new
 
+[FORMAT]
+max-line-length=140
```

Comment:
That’s not compatible with black. Why?

Reply:
Hmm, I wanted to have a bit more space for the documentation, since I am using some of the Overkiz documentation comments.